src/schema/entities.yaml

```diff
@@ -0,0 +1,135 @@
+---
+sub:
+  name: Subject
+  description: |
+    A person or animal participating in the study.
+  format: label
+ses:
+  name: Session
+  description: |
+    A logical grouping of neuroimaging and behavioral data consistent across
+    subjects.
+    Session can (but doesn't have to) be synonymous to a visit in a
+    longitudinal study.
+    In general, subjects will stay in the scanner during one session.
+    However, for example, if a subject has to leave the scanner room and then
+    be re-positioned on the scanner bed, the set of MRI acquisitions will still
+    be considered as a session and match sessions acquired in other subjects.
+    Similarly, in situations where different data types are obtained over
+    several visits (for example fMRI on one day followed by DWI the day after)
+    those can be grouped in one session.
+    Defining multiple sessions is appropriate when several identical or similar
+    data acquisitions are planned and performed on all -or most- subjects,
+    often in the case of some intervention between sessions (e.g., training).
+  format: label
+task:
+  name: Task
+  format: label
+  description: |
+    Each task has a unique label that MUST only consist of letters and/or
+    numbers (other characters, including spaces and underscores, are not
+    allowed).
+    Those labels MUST be consistent across subjects and sessions.
+acq:
+  name: Acquisition
+  description: |
+    The OPTIONAL acq-<label> key/value pair corresponds to a custom label the
+    user MAY use to distinguish a different set of parameters used for
+    acquiring the same modality.
+    For example this should be used when a study includes two T1w images - one
+    full brain low resolution and and one restricted field of view but high
+    resolution.
+    In such case two files could have the following names:
+    sub-01_acq-highres_T1w.nii.gz and sub-01_acq-lowres_T1w.nii.gz, however the
+    user is free to choose any other label than highres and lowres as long as
+    they are consistent across subjects and sessions.
+    In case different sequences are used to record the same modality (e.g. RARE
+    and FLASH for T1w) this field can also be used to make that distinction.
+    At what level of detail to make the distinction (e.g. just between RARE and
+    FLASH, or between RARE, FLASH, and FLASHsubsampled) remains at the
+    discretion of the researcher.
+  format: label
+ce:
+  name: Contrast Enhancing Agent
+  description: |
+    Similarly the OPTIONAL ce-<label> key/value can be used to distinguish
```

Comment:
optional or not is a property per specific modality etc.

"Similarly ..." is the paragraph opening .

`ce-<label>` is duplicate with the record itself.

"can be used to distinguish" is also somewhat "to make it a sentence".

Here and in others I would have just kept it to the point, and start from the 2nd sentence: "The label is the name of the contrast agent.".

Reply:
Do you mind if we push updating entity definitions until a PR for #567? I'm planning to make a version of #568 that is compatible with this PR's changes.

Reply:
I would not mind at all!  Just wanted to leave  a note ;-)